resources/js/visual-editor/editor/components/editor.css

```
@@ -107,3 +107,97 @@
+    gap: 6px;
+    justify-content: flex-end;
+}
+
+.ve-sr-only {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
```

Stylelint (17.6.0)

[error] Unexpected deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)